content/posts/2021-04-19-flank.md

```diff
@@ -10,33 +10,33 @@ I will soon give a live presentation about Flank
 (my first time giving a public presentation + IN JAPANESE).
 I'm sure you can easily imagine my stress right now 😄
 
-In order to fully prepare, I've also decided to also write a blog post about
-my experience using this wonderful tool, allowing us to both reduce our waiting time
-but also bring new opportunities, thanks to the way Flank was implemented.
+In order to fully prepare, I've decided to write a blog post about
+my experience using this wonderful tool Flank, which both reduces the waiting time for running UI tests
+and offers new opportunities, thanks to the way it was implemented.
 
 # Test Lab
 
 If you're already using Android E2E (end-to-end, aka UI tests) tests,
 you already probably know Firebase Test Lab.
 
-It's a Google service allowing to access on-demand virtual or real devices,
```

Comment:
`allowing to` -> Please stop using this phrase 🤣

Reply:
@EmmaKiwi but I like it so much 😆